### ISSUE TITLE
运行run_metro_handmesh.py训练时，出错

### ISSUE THREAD

**Issue by 464hee** [microsoft/MeshTransformer#60]
当   args.num_train_epochs=1时存在训练异常
下面函数
pred_camera, pred_3d_joints, pred_vertices_sub, pred_vertices = METRO_model(images, mano_model, mesh_sampler, meta_masks=meta_masks, is_train=True)

输出pred_camera是一个维度的列表，例如：[1,2,3]
但是visualize_reconstruction需要这个pred_camera参数是一个二维列表，例如：[ [1,2,3] ]

虽然可以将  args.num_train_epochs=2实现解决，但希望能更新

**Comment by PomIsBest** [comment]
我想知道你是否成功下载了训练数据集‘filename.tar’,如果下载成功了，能否共享一下方法？
十分感谢！

**Comment by kevinlin311tw** [comment]
@464hee Good catch! I will look into this problem. I thought it is related to batch size. I am not sure if the number of epochs can also cause the problem.

@PomIsBest See my reply in another thread. Thanks.

**Comment by Bodhi007** [comment]
你好，我在centos上配环境，opendr怎么都安装不上，请问我只跑一跑demo可以不安装opender么

**Comment by 39figthing** [comment]
> 好渔获！我会研究这个问题。我认为这与批量大小有关。我不确定纪元数是否也会导致问题。
> 
> 请参阅我在另一个帖子中的回复。谢谢。

你好，我bash scripts/download_models.sh运行不成功， Azure Blob 存储，禁止公开访问，请问可以将权重文件发送到我的邮箱1060043048@qq.com吗

**Comment by 39figthing** [comment]
> 我想知道你是否成功下载了训练数据集'filename.tar'，如果下载成功了，能否共享一下方法？ 十分感谢！

你好，我bash scripts/download_models.sh运行不成功， Azure Blob 存储，禁止公开访问，请问可以将权重文件发送到我的邮箱1060043048@qq.com吗

**Comment by 39figthing** [comment]
> 当 args.num_train_epochs=1时存在训练异常 下面函数 pred_camera， pred_3d_joints， pred_vertices_sub， pred_vertices = METRO_model（images， mano_model， mesh_sampler， meta_masks=meta_masks， is_train=True）
> 
> 输出pred_camera是一个维度的列表，例如：[1,2,3] 但是visualize_reconstruction需要这个pred_camera参数是一个二维列表，例如：[ [1,2,3] ]
> 
> 虽然可以将 args.num_train_epochs=2实现解决，但希望能更新

你好，我bash scripts/download_models.sh运行不成功， Azure Blob 存储，禁止公开访问，请问可以将权重文件发送到我的邮箱1060043048@qq.com吗

**Comment by 39figthing** [comment]
> 当 args.num_train_epochs=1时存在训练异常 下面函数 pred_camera， pred_3d_joints， pred_vertices_sub， pred_vertices = METRO_model（images， mano_model， mesh_sampler， meta_masks=meta_masks， is_train=True）
> 
> 输出pred_camera是一个维度的列表，例如：[1,2,3] 但是visualize_reconstruction需要这个pred_camera参数是一个二维列表，例如：[ [1,2,3] ]
> 
> 虽然可以将 args.num_train_epochs=2实现解决，但希望能更新

你好，我bash scripts/download_models.sh运行不成功， Azure Blob 存储，禁止公开访问，请问你下载成功了吗？如果可以将权重文件发送到我的邮箱1060043048@qq.com吗

**Comment by 39figthing** [comment]
> 我想知道你是否成功下载了训练数据集'filename.tar'，如果下载成功了，能否共享一下方法？ 十分感谢！

你好，我bash scripts/download_models.sh运行不成功， Azure Blob 存储，禁止公开访问，请问你下载成功了吗？如果可以将权重文件发送到我的邮箱 [1060043048@qq.com](mailto:1060043048@qq.com) 吗
